vector/src/main/java/im/vector/app/features/spaces/manage/SpaceAddRoomFragment.kt

```diff
@@ -139,6 +152,14 @@ class SpaceAddRoomFragment @Inject constructor(
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        withState(viewModel) { state ->
+            val title = if (state.onlyShowSpaces) getString(R.string.space_add_existing_spaces) else getString(R.string.space_add_existing_rooms_only)
```

Comment:
Why we need that? invalidate method should be enough

Reply:
Done